vmdb/lib/workers/schedule_worker.rb

```diff
@@ -233,6 +216,23 @@ def schedules_for_scheduler_role
       @queue.enq :storage_scan_timer
     end
 
+    schedules_for_ems_refresh
+  end
+
+  def schedules_for_ems_refresh
+    config = VMDB::Config.new("vmdb").config.fetch(:ems_refresh, {})
+
+    ExtManagementSystem.leaf_subclasses.each do |klass|
```

Comment:
My point is we care about concrete classes here, not abstract ones that won't typically be instantiated. The fact that they're in a tree doesn't help me. For example, what would happen if we had ConfigurationManager but nothing yet subclassing it yet because reasons... Just because it's a leaf, I don't know that it means it's a concrete class.